ts/screens/wallet/payment/ManualDataInsertionScreen.tsx

```diff
@@ -149,54 +146,42 @@ class ManualDataInsertionScreen extends React.Component<Props, State> {
             </TouchableDefaultOpacity>
 
             <Form>
-              <Item
-                style={styles.noLeftMargin}
-                floatingLabel={true}
-                error={this.state.paymentNoticeNumber
-                  .map(isLeft)
-                  .getOrElse(false)}
-                success={this.state.paymentNoticeNumber
-                  .map(isRight)
-                  .getOrElse(false)}
-              >
-                <Label>{I18n.t("wallet.insertManually.noticeCode")}</Label>
-                <Input
-                  keyboardType={"numeric"}
-                  returnKeyType={"done"}
-                  maxLength={18}
-                  onChangeText={value => {
+              <LabelledItem
```

Comment:
It has a different behaviour with respect to the master branch.
| Before | After|
|-------|-------|
|<img src="https://user-images.githubusercontent.com/11773070/126490516-aea0707a-bed5-4139-b19a-5d3156f3a8ca.png" width=200>|<img src="https://user-images.githubusercontent.com/11773070/126490479-5b5f862d-0598-4b8d-89b0-bb89def4ce25.png" width=200>|

Reply:
Yes, it's correct. I talked about this case with @thisisjp  and he confirmed that this screen has an old layout, it would be more correct to introduce the new changes also in this screen so as to have the same layout for the text fields.

Reply:
@andrea-favaro could you add some space between the "Dove si trovano?" and the forms? We'll change the rest in the future ;)